content/departments/engineering/teams/growth/cloud-trial-operations.md

```diff
@@ -0,0 +1,106 @@
+# Cloud trial operations
+
+From 2022-09-27, Sourcegraph Cloud will be the default way of deploying Sourcegraph. All the call to actions (CTAs) on Sourcegraph sites and posts will let users request a 30-day cloud trial. This page describes the workflow and expectations from each team to make sure those trial users get a great experience. It also describes what the role of each team will be in the longer term.
+
+## Context
+
+Cloud is the default way of deploying Souregraph, but there are some limits to delivering a cloud trial instance today.
+
+- There is a cap on the number of instances we can managed at any given time ([SLA](#TODO)). Therefore, when someone requests an instance, we have to qualify them to make sure that we have capacity and that a cloud instance is the best option to meet their needs.
+- Cloud instances take some time to provision ([SLA](#TODO)), that makes it so signing up for a cloud instance is not instance.
```

Comment:
This is pending this question: https://sourcegraph.slack.com/archives/C03JR7S7KRP/p1663789661106919